vitrina/templates/component/tabs.html

```diff
@@ -0,0 +1,18 @@
+{% load i18n %}
+
+<div class="tabs">
+    <ul>
+        <li class="{%if request.resolver_match.url_name == detail_url_name %} is-active {% endif %}">
+            <a href="{% url detail_url_name request.resolver_match.kwargs.pk %}">
+                {% translate "Informacija" %}
+            </a>
+        </li>
+        {% if can_manage_history %}
+            <li class="{%if request.resolver_match.url_name == history_url_name %} is-active {% endif %}">
+                <a href="{% url history_url_name request.resolver_match.kwargs.pk %}">
+                    {% translate "Istorija" %}
+                </a>
+            </li>
```

Comment:
This is not shown for projects, with `user.is_staff = True`.

Reply:
I tried to repeat this with user that is not superuser, but only staff, seems to work. Also I added follow() to tests, history tab is available there too.